### PR TITLE
Add popup window

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I;
- Created the project popup modal
- created the different elements dynamically for each project
- Added the cards dynamically to the main container .projects
- Fixed linters issues

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+
+.popup-modal {
+  position: absolute;
+  top: 0%;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--image-placeholder-color);
+  z-index: 100;
+}
```
Comment (emmiiorji) [inline assets/css/styles.css:761]: Kindly adjust the design of the popup to be as close to the design as possible. The background of the popup is a blur of the page and not given a solid color.

Your design:
image

Figma design:
image

File: assets/css/styles.css [hunk 2]
```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
```
Comment (emmiiorji) [inline assets/css/styles.css:799]: In order to maintain a cleaner code base, kindly remove all commented codes.

File: assets/css/styles.css [hunk 3]
```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+}
+
+.js-liveLink-button {
+  /*
+  width: 20%;
+  height: 20%;
+  */
+  background: url(../../images/popup/button-see-live.svg) no-repeat center;
+  background-size: 100%;
+}
```
Comment (emmiiorji) [inline assets/css/styles.css:885]: Kindly ensure that the live and GitHub links show up as represented by the design. This also applies to the languages and description.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi Team, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Hardcoded project items have been removed
- Popup renders and closes as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diarisdiakite** [comment]
In this PR, I :
- blurred the background;
- resolved th button issue;
- tried several solutions to get my technologies list styled.

File: assets/css/styles.css [hunk 4]
```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
```
Comment (roshan-bajgain) [inline assets/css/styles.css:812]: -[ ] As requested by the previous reviewer please make sure to remove this commented code as it makes does not make the clean project.

File: assets/css/styles.css [hunk 5]
```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+  background: url(../../images/popup/icon-gitHub.svg) no-repeat center;
+}
+
+.js-liveLink-button {
+  /*
+  width: 20%;
+  height: 20%;
+  */
+  background: url(../../images/popup/button-see-live.svg) no-repeat center;
+  background-size: 100%;
```
Comment (roshan-bajgain) [inline assets/css/styles.css:897]: - [ ] As requested by the previous reviewer please make your design similar to the Figma design as buttons and descriptions are not showing on the pop-up window. See the image below for reference
- [ ] When I scroll then blur part is scrolling and the buttons and description is not showing please work on it and make sure to match the design.
### Figma design
fimga design

### Your design
his

File: assets/css/styles.css [hunk 6]
```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
...
+  left: 0;
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+  z-index: 100;
+  backdrop-filter: blur(15px);
+  backdrop-filter: brightness(60%);
+  backdrop-filter: contrast(40%);
+}
```
Comment (roshan-bajgain) [inline assets/css/styles.css:783]: - [ ] [Optional] Please stick to the design as much as possible. Please make your mobile design a Figma design.
- [ ] [Optional] Please stick to the design like color, buttons and `X` buttons, spacing etc.
### Your design
his

### Figma design
fimga

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @diarisdiakite, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good job on required changes ✔️ 
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/styles.css (hunk 1 above)
Comment (diarisdiakite) [reply assets/css/styles.css]: @done

File: assets/css/styles.css (hunk 2 above)
Comment (diarisdiakite) [reply assets/css/styles.css:812]: Done @emmiiorji

File: assets/css/styles.css (hunk 3 above)
Comment (diarisdiakite) [reply assets/css/styles.css:898]: Button fixed @emmiiorji

File: assets/css/styles.css (hunk 4 above)
Comment (diarisdiakite) [reply assets/css/styles.css:812]: Done @roshan-bajgain

File: assets/css/styles.css (hunk 5 above)
Comment (diarisdiakite) [reply assets/css/styles.css:897]: Done @roshan-bajgain 
Image height fixed

**Comment by diarisdiakite** [comment]
In this milestone, I:
- Fixed the image height,
- fix the mobile close image to fit the Figma template
- added a border radius to my popup

File: assets/js/projectCards.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+          </h3>
+        </div>
+        <ul class="project-built-with" name="technologies">
+          <li><a href="tonhttps://rubyonrails.org/" rel="noopener" target="_blank">${project.technologies[0]}</a></li>
+          <li><a href="https://www.w3.org/Style/CSS/Overview.en.html" rel="noopener" target="_blank">${project.technologies[1]}</a></li>
+          <li><a href="https://www.javascript.com/" rel="noopener" target="_blank">${project.technologies[2]}</a></li>
+          <li><a href="https://html.com/" rel="noopener" target="_blank">${project.technologies[3]}</a></li>
+        </ul>
+        <button class="project-buttons" id="projectPopup">See project</button>
+      </div>
```
Comment (V-Blaze) [inline assets/js/projectCards.js:110]: - [OPTIONAL] Great job implementing the `See Live` and `See source` buttons. :smile: :+1:  But I think you could make them follow the rest of the design by implementing the button states styles `hover`, `active` :+1:

**Review by V-Blaze — APPROVED** [review]
Hi @diarisdiakite @JuanDBta  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights

- You implemented all the suggested changes from the previous reviewers 💯
-  Your pop-up works fine both on mobile and desktop👍
- Excellent interaction and communication with the reviewers 🟢 👍

## Optional suggestions

- [OPTIONAL] Please go through your `popup.js` file and remove all the unused code left as comments, This will make your codebase clean and also follow the best coding practices. :wave: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**